projects/micro-dash/src/lib/lang/is-match.spec.ts

```diff
@@ -2,6 +2,16 @@ import { isMatch } from "./is-match";
 import { Falsey, ObjectWith } from "../interfaces";
 
 describe("isMatch()", () => {
+  // hits some code coverage missed by the lodash tests
+  it("can tell the difference between an object and an array", () => {
+    expect(isMatch<any>(["a"], { 1: "a" })).toBe(false);
```

Comment:
I got an email notification about this.. sorry to be snooping. But wouldn't these not match anyway even if they were both objects or both arrays? I would expect the case that might possibly be wrong to be more like `isMatch(['a'], { 0: 'a' })`.

Reply:
You snoop! Stop spontaneously improving my code!